wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -73,6 +73,7 @@ public Long getRobotMainThreadId() {
   }
 
   protected final DriverStation m_ds;
+  private static boolean m_hardwareInitialized = false;
```

Comment:
Got a PMD complaint:
wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java:76:    Avoid using redundant field initializer for 'm_hardwareInitialized'
Removing the '= false' eliminated it.